### PR TITLE
Initial commit for Toast Swift Package

### PR DESCRIPTION
**Pull request by maor-loora**
I Know it's targeted main->for_first_pr, but as main already exist that's the only way to make a normal PR, so I've created any for any comment that might exist. once approval I won't merge it and remove this branch (code will be kept on main)

### CODE REVIEW

File: Sources/Toast/ToastModifier.swift [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        self.hidingTimer?.invalidate()
+        self.hidingTimer = nil
+    }
+    
+    private func handleGesture(isDownDirection: Bool) {
+        guard self.configuration.dragToDismiss else {
+            return
+        }
+        
+        switch self.configuration.direction {
```
Comment (ameedsayeh) [inline Sources/Toast/ToastModifier.swift:113]: I think we miss stopping the timer ?

Comment (ameedsayeh) [reply Sources/Toast/ToastModifier.swift:113]: oh nvm you handle it inside onChange

**Review by ameedsayeh — APPROVED** [review]
👍🏼  nice one! I have a small suggestion to keep the readme file short and straight to the point

**Comment by maor-loora** [comment]
Closed, as It is already in main and no changes.

**Pull request closed without merging**